lib/routes/account.js

```diff
@@ -972,6 +979,7 @@ module.exports = function (
       },
       handler: function (request, reply) {
         log.begin('Account.devices', request)
+        var language = request.query.language || 'en'
```

Comment:
Should we default to using current accept-language header, or will that just risk inconsistency between the language chosen here vs in content-server?